lib/realtime_web/channels/user_socket.ex

```diff
@@ -144,4 +166,39 @@ defmodule RealtimeWeb.UserSocket do
   end
 
   defp max_heap_size(), do: Application.fetch_env!(:realtime, :websocket_max_heap_size)
+  defp measure_traffic_interval_in_ms(), do: Application.fetch_env!(:realtime, :measure_traffic_interval_in_ms)
+
+  defp collect_traffic_telemetry(nil, _tenant_external_id, prev_bytes), do: prev_bytes
+
+  defp collect_traffic_telemetry(transport_pid, tenant_external_id, prev_bytes) do
+    transport_pid
+    |> Process.info(:links)
+    |> then(fn {:links, links} -> links end)
```

Comment:
Do we expect more than 1 port being here? If we do we would need to track prev bytes per port then. I wonder if we should always just get 1 port here and not call reduce so that's explicit that we only want to account for 1 TCP connection and if one day this returns more than 1 port we probably want to account for multiple ports. 

Another option is to accumulate the new bytes and with the result of `Enum.reduce` we call telemetry with the difference and return the result of Enum.reduce as new values. I hope this makes sense

Reply:
I don't expect it, it's to prevent it from blowing up